pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
+    return x / y
```

Comment:
I think the correct calculation would be
```suggestion
    return x / z
```